python/doc/theory/data_analysis/anderson_darling_test.rst

```diff
@@ -1,68 +1,60 @@
+distribution.
+
+This test involves the calculation of the test statistic which is
+the distance between the empirical cumulative distribution function
+:math:`F_{\sampleSize}` and :math:`\Phi`. Letting :math:`X_1, \ldots , X_{\sampleSize}`
+be independent random variables respectively distributed according to :math:`F`, we define
+the the order statistics :math:`X_{(1)}, \ldots , X_{(\sampleSize)}` by:
 
 .. math::
 
-   \begin{aligned}
-       D = \int^{\infty}_{-\infty} \frac{\displaystyle \left[F\left(x\right) - \widehat{F}_N\left(x\right)\right]^2 }{\displaystyle F(x) \left( 1-F(x) \right) } \, dF(x)
-     \end{aligned}
+   X_{(1)} \leq \dots \leq X_{(\sampleSize)}
 
-With a sample :math:`\left\{ x_1,x_2,\ldots,x_N \right\}`, the distance
-is estimated by:
+The test statistic is defined by:
+
+.. math::
+
+       D_{\sampleSize} = -\sampleSize-\sum^{\sampleSize}_{i=1} \frac{2i-1}{\sampleSize} \left[\log \Phi(X_{(i)})+\log\left(1-\Phi(X_{(\sampleSize+1-i)})\right)\right]
+
+This distance is a quadratic
+type, as in the :ref:`Cramer-Von Mises test <cramer_vonmises_test>`,
+but gives more weight to deviations of tail values. The empirical value of the test statistic denoted by :math:`d_{\sampleSize}` is evaluated from the sample sorted in ascending order:
 
 .. math::
 
-   \begin{aligned}
-       \widehat{D}_N = -N-\sum^{N}_{i=1} \frac{2i-1}{N} \left[\ln F(x_{(i)})+\ln\left(1-F(x_{(N+1-i)})\right)\right]
-     \end{aligned}
+   x_{(1)} \leq \dots \leq x_{(\sampleSize)}
 
-where :math:`\left\{x_{(1)}, \ldots, x_{(N)}\right\}` describes the
-sample placed in increasing order.
+Under the null hypothesis :math:`\mathcal{H}_0 = \{ F = \Phi\}`, the distribution of the test statistic :math:`D_{\sampleSize}` is
+asymptotically known i.e. when :math:`\sampleSize \rightarrow +\infty`.
+If :math:`\sampleSize` is sufficiently large, we can use the asymptotic distribution to apply the test
+as follows.
+We fix a risk :math:`\alpha` (error type I) and we evaluate the associated critical value :math:`d_\alpha` which is the quantile of order
+:math:`(1-\alpha)` of :math:`D_{\sampleSize}`.
 
-The probability distribution of the distance :math:`\widehat{D}_N` is
-asymptotically known (i.e. as the size of the sample tends to infinity).
-If :math:`N` is sufficiently large, this means that for a probability
-:math:`\alpha` and a candidate distribution type, one can calculate the
-threshold / critical value :math:`d_\alpha` such that:
+Then a decision is made, either by comparing the test statistic to the theoretical threshold :math:`d_\alpha`
+(or equivalently
+by evaluating the p-value of the sample  defined as :math:`\Prob{D_{\sampleSize} > d_{\sampleSize}}` and by comparing
+it to :math:`\alpha`):
 
--  if :math:`\widehat{D}_N>d_{\alpha}`, we reject the candidate
-   distribution with a risk of error :math:`\alpha`,
+-  if :math:`d_{\sampleSize}>d_{\alpha}` (or equivalently :math:`\Prob{D_{\sampleSize} > d_{\sampleSize}} \leq \alpha`),
```

Comment:
```suggestion
-  if :math:`d_{\sampleSize}>d_{\alpha}` (or equivalently :math:`\Prob{D_{\sampleSize} > d_{\sampleSize}} < \alpha`),
```

Reply:
The test statistics is continuous. Then there is no difference between < and $\leq$.

Reply:
I understand that the probability of getting exactly $\alpha$ is zero, because the statistics is a continuous variable. This is, however, a question of logic: the alternative must be the logical opposite.